doc/authors.rst

```diff
@@ -1,17 +1,36 @@
+.. _chapter_authors:
 
-
-=======
 Authors
 =======
 
-- Andreas J. Thorvaldsen
-- Arnfinn H. Steindal
-- Bin Gao
-- Dan Jonsson
-- Daniel Friese
-- Jógvan Magnus Haugaard Olsen
-- Kenneth Ruud
-- Magnus Ringholm
-- Michal Repisky
-- Radovan Bast
-- Thomas Kjærgaard
+The main developing team of OpenRSP program and previous authors (if their
+contributions are still used) consist of (alphabetical order of surname):
+
+.. tabularcolumns:: |p{0.1\textwidth}|p{0.5\textwidth}|p{0.4\textwidth}|
+.. list-table:: OpenRSP authors
+   :header-rows: 1
+
+   * - Name
+     - Affiliation
+     - Email
+   * - Radovan Bast
+     - UiT The Arctic University of Norway
+     - radovan.bast@uit.no
+   * - Daniel H. Friese
+     - UiT The Arctic University of Norway
+     - daniel.h.friese@uit.no
+   * - Bin Gao
+     - UiT The Arctic University of Norway
+     - bin.gao@@uit.no
```

Comment:
Double "@@" - also do we need to list email addresses? It could invite spam harvesters.